internal/common/db.go

```diff
@@ -256,6 +272,47 @@ func AutoMigrate(db *gorm.DB) error {
 	)
 }
 
+func GetOpenshiftVersionsSupportLevelsFromDBWhere(db *gorm.DB, where ...interface{}) ([]*OpenshiftVersionSupportLevel, error) {
+	dbOpenshiftVersionsSupportLevel := []*OpenshiftVersionSupportLevel{}
+	if err := db.Model(&OpenshiftVersionSupportLevel{}).Find(&dbOpenshiftVersionsSupportLevel, where...).Error; err != nil {
+		return nil, err
+	}
+
+	return dbOpenshiftVersionsSupportLevel, nil
+}
+
+func GetReleaseImagesFromDBWhere(db *gorm.DB, where ...interface{}) (models.ReleaseImages, error) {
+	releaseImages := models.ReleaseImages{}
+	dbReleaseImages := []*ReleaseImage{}
+	if err := db.Model(&ReleaseImage{}).Find(&dbReleaseImages, where...).Error; err != nil {
+		return nil, err
+	}
+
+	for _, releaseImage := range dbReleaseImages {
+		cpuArchitectures := []string{*releaseImage.CPUArchitecture}
+		if *releaseImage.CPUArchitecture == MultiCPUArchitecture {
+			cpuArchitectures = SupportedMultiArchitectures
+		}
+		releaseImage.CPUArchitectures = cpuArchitectures
```

Comment:
I agree it is not optimal, but unfortunately I didn't find any way to obtain this information dynamically. I looked into the builds and saw that since `4.11` we are supporting this 4 architectures for every multi release, do you have an idea how to go around it?